internal/core/src/query/SearchBruteForce.cpp

```diff
@@ -35,6 +42,9 @@ raw_search(MetricType metric_type,
            float* D,
            idx_t* labels,
            const BitsetView bitset) {
+#ifdef __APPLE__
+    // TODO
```

Comment:
please make sure binary search whether support force search ,  UT passed but I can't find some clue. @soothing-rain

Reply:
Per our offline discussion, we are not supporting binary distance search in Mac in Milvus 2.1 as there is too much effort. Also binary distance search has very few active users.